changelog/12185.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+secrets/aws: Refactor genUsername tests into table tests
```

Comment:
If we change to errors instead of warnings for custom username templates char limits, we should call it out in here.

Reply:
Updated